README.md

```diff
@@ -16,17 +16,17 @@ randomize keys/certificates used for issuance.
 ## Goals
 
 1. Produce a simplified testing front end
-2. Move rapidly to gain [ACME draft-04](https://tools.ietf.org/html/draft-ietf-acme-acme-04) experience
+2. Move rapidly to gain [ACME draft-06](https://tools.ietf.org/html/draft-ietf-acme-acme-06) experience
 3. Write "idealized" code that can be adopted back into Boulder
 4. Aggressively build in guardrails against non-testing usage
 
 Pebble aims to address the need for ACME clients to have an easier to use,
 self-contained version of Boulder to test their clients against while developing
-ACME-04 support. Boulder is multi-process, requires heavy dependencies (MariaDB,
+ACME-64 support. Boulder is multi-process, requires heavy dependencies (MariaDB,
```

Comment:
Oops! Editor failure. I'm glad we're not on the 64th draft :skull_and_crossbones: